pyocd/debug/svd/__init__.py

```diff
@@ -17,28 +17,17 @@
 
```

Comment:
Would you mind moving the definitions in this file to a `pyocd.debug.svd.loader` module? (Or whatever name you think is best.) I like to keep the `__init__.py` either empty, or only importing symbols from other modules in the package.

Reply:
Done!